Kata RPG/Kata RPG/Warrior.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Warrior.swift
+//  Kata RPG
+//
+//  Created by norm on 07/11/2020.
+//
+
+import Foundation
+
+class Warrior: Character {
+   
+    override func attack(target: Character){
+            if(self.lifeStatus == true){
+                target.health -= Int.random(in: 0...9)
+                if (target.health < 0) {
+                    target.health = 0
+                    target.lifeStatus = false
+                }
+            }
+            else{
+            print("\(self.name) is dead and can't attack")
+            }
+        }
+    
```

Comment:
See comments for the other classes here. You have duplicated code, you should be able to refacto